docs/guides/getting-started-with-bolt.md

```diff
@@ -158,19 +173,25 @@ The default constructor expects the following two env variables exist when start
 
 If you prefer configuring an **App** in a different way, write some code to initialize **AppConfig** on your own.
 
-Anyway, set the two env variables and hit `gradle run` on your terminal. The command runs your main method. For more detailed logging, `gradle run -DslackLogLevel=debug` is also available.
+Set the two environment variables and run:
+
+- Gradle users: `gradle run` (for more detailed logging: `gradle run -DslackLogLevel=debug`),
+- Maven users: `mvn clean compile package exec:java -Dexec.mainClass="hello.MyApp"` (for more detailed logging you can also provide the `-Dorg.slf4j.simpleLogger.defaultLogLevel=debug` flag)
+
+The command runs your main method.
 
 ```bash
 # Visit https://api.slack.com/apps to know these
 export SLACK_BOT_TOKEN=xoxb-...your-own-valid-one
 export SLACK_SIGNING_SECRET=123abc...your-own-valid-one
 
 # run the main function
+# gradle users should run:
 gradle run
+# maven users should run:
+mvn clean compile package exec:java -Dexec.mainClass="hello.MyApp"
 ```
 
-You will see the message saying "**⚡️ Bolt app is running!**" in stdout.
```

Comment:
I removed this as there is no such log line in the example app code, but maybe instead of removing this we add this log line to the example app?

Reply:
@filmaj Can you revert this change for the HTTP mode app? It prints this log line: https://github.com/slackapi/java-slack-sdk/blob/v1.22.1/bolt-jetty/src/main/java/com/slack/api/bolt/jetty/SlackAppServer.java#L131